### DISCUSSION TITLE
Ignore firenvim

### DISCUSSION THREAD

**Discussion by semi710** [olimorris/persisted.nvim#77] [Q&A] [answered]
How to ignore files opened via firenvim plugin in browser?

**Comment by olimorris** [comment]
Unless they have a specific filetype this could be impossible. There are callbacks you could look at which you could enact prior to writing the session to disk.

**Reply by semi710** [reply]
https://github.com/glacambre/firenvim/discussions/1544#discussioncomment-6300951

Check out here the dev mentions that files are created in `$XDG_RUNTIME_DIR/firenvim`.
ignorance of dir is possible?

**Reply by olimorris** [reply, marked as answer]
You could build this in then using the examples in the readme. You'll just need to determine where the buffer exists on disk

**Reply by semi710** [reply]
```lua
{
    "olimorris/persisted.nvim",
    config = function()
      local rtimedir = os.getenv("XDG_RUNTIME_DIR")
      require("persisted").setup({
        ignored_dirs = {
          rtimedir .. "/firenvim",
        },
      })
    end,
  },
```

This works for me in lazy.nvim
